raiden-dapp/tests/unit/views/backup-state.spec.ts

```diff
@@ -44,6 +47,14 @@ describe('BackupState.vue', () => {
     expect(downloadStateDialog).toBeTruthy();
   });
 
+  test('download state button disabled if disconnected', () => {
+    const downloadStateButton = wrapper.find(
+      '.backup-state__buttons__download-state'
+    );
+
+    expect(downloadStateButton.classes()).toContain('v-list-item--disabled');
+  });
+
   // test('upload state', () => {
```

Comment:
I know it's unrelated to the PR but any idea why we have a commented test here?

Reply:
Probably the upload/download state ticket was too big and got split up
They will be implemented in #1308

Reply:
Ok but it would be nice to avoid commented code in the repo in the future :)